src/compass/utils/geo_metadata.py

```diff
@@ -265,3 +265,181 @@ def to_file(self, dst, fmt:str):
             json.dump(self_as_dict, dst, indent=4)
         else:
             raise ValueError(f'{fmt} unsupported. Only "json" or "yaml" supported')
+
+    def to_hdf5(self, dst_h5):
+        '''Write self to HDF5
+
+        Parameter:
+        ---------
+        dst_h5: h5py File
+            HDF5 file meta data will be written to
+        '''
+        metadata_group = dst_h5.require_group('metadata')
+
+        def add_dataset_and_attrs(group, name, value, attr_dict):
+            '''Write isce3.core.Poly1d properties to hdf5
+
+            Parameters
+            ----------
+            group: h5py.Group
+                h5py Group to store poly1d parameters in
+            name: str
+                Name of dataset to add
+            value: object
+                Value to be added
+            attr_dict: dict[str: object]
+                Dict with attribute name as key and some object as value
+            '''
+            group[name] = value
+            val_ds = group[name]
+            for key, val in attr_dict:
+                val_ds.attrs[key] = val
+
+        # subset of burst class attributes
+        add_dataset_and_attrs(metadata_group, 'sensing_start',
+                              np.string_(self.sensing_start.__str__),
+                              {'description':'azimuth sensing start'})
```

Comment:
We are not specifying the units here, I am not sure if we should specify the format as here https://docs.google.com/document/d/1IUaMs4HYYKIuUiQ6edFo-srDxTOHL62n/edit